scout/server/blueprints/institutes/controllers.py

```diff
@@ -431,6 +439,20 @@ def cases(store, request, institute_id):
     limit = int(request.args.get("search_limit")) if request.args.get("search_limit") else 100
     data["form"] = populate_case_filter_form(request.args)
 
+    ALL_CASES_PROJECTION = {
```

Comment:
Very nice! But what about the fields that should be in the "share" column of cases page?

Reply:
Thanks, I'll doublecheck - pretty sure I saw the icon, but maybe that was a different version. 🙄

Reply:
I remembered correctly at least:
![Screenshot 2023-10-23 at 16 58 08](https://github.com/Clinical-Genomics/scout/assets/758570/2a4d5076-131d-46fc-a4d3-abbab9cc3b71)
But, Ill check if Im missing any of them!